commands/qx.go

```diff
@@ -175,14 +178,32 @@ var pritoaddrCmd = &cobra.Command{
 	},
 }
 
-var wifToPriCmd = &cobra.Command{
-	Use:   "wiftopri {string, wif}",
-	Short: "WIF key to private key",
-	Example: `
+
+func newWifToPriCmd()*cobra.Command{
+	var format string
+	var wifToPriCmd = &cobra.Command{
+		Use:   "wiftopri {string, wif}",
+		Short: "WIF key to private key",
+		Example: `
 		wiftopri  5HueCGU8rMjxEXxiPuD5BDku4MkFqeZyd4dZ1jvhTVqvbTLvyTJ
+		wiftopri  PxBefLecRtTYPoxUUwAbq8m7xGmzDK8gQ71N9qKyhp2j5yN42Rpzc --format=wallet_v0.9 --network=mixnet
```

Comment:
where is the flag handing for  the `--network` option ? it looks only bind the flag `--format` to the `var format`.

Reply:
Only '--format=wallet_v0.9' requires the network option, and private keys starting with 5H do not have the netwaork option.